config/vufind/DAIA.ini

```diff
@@ -56,6 +56,10 @@ daiaContentTypes['xml'] = "application/xml"
 ; daiaContentTypes['json'] = "application/json, application/javascript"
 daiaContentTypes['json'] = "application/json"
 
+; accepted on-site services
+; (separate multiple values by colons)
+daiaOnSiteServices = loan:presentation:interloan
```

Comment:
As the default for onSiteServices in DAIA.php is loan and presentation, I guess this should be reflected here as well. So I would not put interloan into the configuration setting by default.

Reply:
You are right; I fixed it.

Reply:
Thank you :-)